crates/test-collector/src/lib.rs

```diff
@@ -273,6 +309,117 @@ fn extract_panic_values(db: &dyn SyntaxGroup, attr: &Attribute) -> Option<Vec<Fe
         .collect::<Option<Vec<_>>>()
 }
 
+/// Tries to extract the fork configuration.
+fn extract_fork_config(db: &dyn SyntaxGroup, attr: &Attribute) -> Option<Fork> {
```

Comment:
This will probably fail silently if something is wrong with the configuration, right? Perhaps we should display some diagnostic info

Reply:
Yup, I thought about that but didn't have time to develop suitable solution

Reply:
Perhaps we should just use unwraps/expects here in case configuration cannot be loaded? So it fails in an more explicit way?

Reply:
I added changes and it fork behaves exactly the same should_panic in the context of fail